### PR TITLE
templates: addition of ATLAS news item

### PR DESCRIPTION
**Pull request by AnxhelaDani**
Signed-off-by:  Sunje Dallmeier-Tiessen sunje.dallmeier-tiessen@cern.ch

### CODE REVIEW

File: invenio_opendata/base/templates/data/news_db.html [hunk 1]
```diff
@@ -1,5 +1,19 @@
 {% set news = [
   (
+  'Explore ATLAS Open Data resources', '
+  '<p>
+    Starting today, the data from 100 trillion proton collisions is now public on the Open Data Portal)! This marks the world's first open release of 8 TeV data, gathered from the Large Hadron Collider in 2012.
+  </p>
+  <p>
+    Unique to the ATLAS Open Data release is its strong emphasis on learning. Hence, additional resources are available via the ATLAS educational platform which guides users through the data, explains how to visualize and how to download and use the data, and even provides open-source software. As part of this, ATLAS has made seven physics analyses available for users to get started with their research. By releasing additional resources and comprehensive documentation ATLAS aims at bridging the gap between phycisists and the public.
```
Comment (RaoOfPhysics) [inline invenio_opendata/base/templates/data/news_db.html]: Is it really unique to ATLAS? :P 

Anyway, spelling should be -ise, not -ize. So, "visualise".

Comment (tiborsimko) [inline invenio_opendata/base/templates/data/news_db.html]: > Is it really unique to ATLAS?

:+1: 

What about reformulating like "A notable aspect of this ATLAS open data release is..."

File: invenio_opendata/base/templates/data/news_db.html [hunk 2]
```diff
@@ -1,5 +1,19 @@
 {% set news = [
   (
+  'Explore ATLAS Open Data resources', '
```
Comment (tiborsimko) [inline invenio_opendata/base/templates/data/news_db.html]: Note the extra quote at the end of the line that is not needed.

File: invenio_opendata/base/templates/data/news_db.html (hunk 1 above)
Comment (RaoOfPhysics) [inline invenio_opendata/base/templates/data/news_db.html]: Or:

> Given its strong emphasis on learning, the release of ATLAS Open Data is accompanied by additional resources that are available via the ATLAS educational platform. The resources guide users through the data…

?

**Comment by suenjedt** [comment]
@RaoOfPhysics sounds good to me

**Comment by AnxhelaDani** [comment]
Done! Thank you @RaoOfPhysics  & @tiborsimko :)

**Comment by tiborsimko** [comment]
Thanks, I'll squash, there is also a parenthesis and a quote problem, I'll fix that during merge.

**Pull request closed without merging**